ui/package.json

```diff
@@ -71,9 +73,9 @@
     "postcss": "^8.5.3",
     "prettier": "^3.5.3",
     "prettier-plugin-tailwindcss": "^0.6.11",
-    "tailwindcss": "^3.4.17",
+    "tailwindcss": "^4.1.6",
```

Comment:
Major upgrade, changed the Menu 
`Menu.Button` -> `MenuButton`
`Menu.Items` -> `MenuItems` 
`Menu.Item` -> `MenuItem`

Additionally the menu no longer works with `Card` embedded as the parent for the items, so restructured the Header.tsx